core-tests/src/test/java/org/mule/runtime/core/internal/config/DefaultFeatureFlaggingServiceTestCase.java

```diff
@@ -7,79 +7,84 @@
 
 package org.mule.runtime.core.internal.config;
 
-import io.qameta.allure.Feature;
 import io.qameta.allure.Story;
-import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 import org.junit.runners.Parameterized.Parameters;
+import org.mule.runtime.api.config.Feature;
 import org.mule.runtime.api.exception.MuleRuntimeException;
 import org.mule.runtime.core.api.config.FeatureFlaggingService;
 
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.function.Consumer;
+import java.util.function.Supplier;
 
 import static java.lang.String.format;
 import static java.util.Arrays.asList;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
+import static org.mule.runtime.api.config.Feature.TESTING_FEATURE;
 import static org.mule.test.allure.AllureConstants.DeploymentConfiguration.DEPLOYMENT_CONFIGURATION;
 import static org.mule.test.allure.AllureConstants.DeploymentConfiguration.FeatureFlaggingStory.FEATURE_FLAGGING;
 
 @RunWith(Parameterized.class)
-@Feature(DEPLOYMENT_CONFIGURATION)
+@io.qameta.allure.Feature(DEPLOYMENT_CONFIGURATION)
```

Comment:
why?

Reply:
We need to reference both our enum and the allure Feature annotation. One of them has to have a fully qualified class name.